extension/services/youtube/youtube.js

```diff
@@ -23,8 +23,17 @@ async function waitForUrl(tabId, options) {
 }
 
 class YouTube extends serviceList.Service {
+  async getYoutubeTabId() {
+    const youtubeTabs = await this.getAllTabs({ currentWindow: true });
```

Comment:
We don't want to reuse a background tab. I think this could probably be fixed just by adding `active: true` to the `getAllTabs` query.